src/styles/pages/_index.scss

```diff
@@ -546,7 +546,7 @@ div.gbp {
 }
 
 .sent-email-notification {
-  font-size: 25px;
+  font-size: 1.5em;
   margin: 50px 0 0 7px;
```

Comment:
Outside of known sizes for the screen / format, I don't think so. What happens if someone visits with a very different number of pixels on their screen than you?

I know that CSS pixels are not exactly device pixels, but this still seems like a problem to me